pootle/apps/pootle_store/unit/filters.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+class UnitTextSearch(object):
+    """Search Unit's fields for text strings
+    """
+
+    search_fields = (
+        "source_f", "target_f", "locations",
+        "translator_comment", "developer_comment")
+    search_mappings = {
+        "notes": ["translator_comment", "developer_comment"],
+        "source": ["source_f"],
+        "target": ["target_f"]}
+
+    def __init__(self, qs):
+        self.qs = qs
+
+    def get_search_fields(self, sfields):
+        search_fields = set()
+        for field in sfields:
+            if field in UnitTextSearch.search_mappings:
```

Comment:
No. It is just that I find confusing referring to its own class using the full class name, so converting to `classmethod` would allow to use `cls` which is shorter and less confusing to me. But it is just a nit.